src/include/robin_hood.h

```diff
@@ -220,7 +220,7 @@ T* assertNotNull(T* t, Args&&... args) {
 }
 
 template <typename T>
-inline T unaligned_load(void const* ptr) {
+inline T unaligned_load(void const* ptr) noexcept {
```

Comment:
Shouldnt it be `noexcept(noexcept(std::memcpy))` to be save here?

Reply:
`std::memcpy` will never throw an exception. It's essentially a C function in a C++ namespace. It might segfault, but that's a different beast altogether.